src/components/Charts/Charts.tsx

```diff
@@ -22,17 +21,27 @@ export default function Charts({
 }) {
 	// Definitions
 	const [chartData, setChartData] = React.useState<Array<{}>>([{}]);
+	const [dataToDisplay, setDataToDisplay] = React.useState<{
+		[group: string]: boolean;
+	}>({
+		...configGroups.reduce(
```

Comment:
Aussi valable ici de ne pas utiliser reduce pour cette utilisation.

Reply:
Removed in future PR.